urlchecker/client/check.py

```diff
@@ -106,9 +106,10 @@ def main(args, extra):
         if args.verbose:
             print("\n\U0001F914 Uh oh... The following urls did not pass:")
             for file_name, result in checker.checks.items():
-                print_failure(file_name + ":")
-                for url in result.failed:
-                    print_failure("     " + url)
+                if result.failed != []:
```

Comment:
Here you can usually just do:

```
if result.failed:
```
That will check for empty list, length of 0, and the value 0 so it should only print if there are entries in the list.